pom.xml

```diff
@@ -17,7 +17,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <common.version>1.0.9</common.version>
+        <common.version>1.0.11</common.version>
```

Comment:
Update always to the latest one if it needs to be updated

Reply:
Will upgrade to the latest version of common...